qa/acceptance/spec/debian/spec_helper.rb

```diff
@@ -0,0 +1,20 @@
+require 'runner-tool'
+require_relative '../../../rspec/helpers'
+require_relative '../../../rspec/matchers'
+require_relative '../config_helper'
+
+def vagrant_boxes
+  [
+    "ubuntu-1204",
+    "ubuntu-1404"
+  ]
+end
+
+RunnerTool.configure
+SpecsHelper.configure(vagrant_boxes)
+
+puts "[Acceptance specs] running on #{ServiceTester.configuration.lookup.values}"
+
+RSpec.configure do |c|
+  c.include ServiceTester
+end
```

Comment:
This code is almost the same as https://github.com/elastic/logstash/pull/5138/files#diff-560541d08dfcd2f1980e7a56c37e1b3eR1 we should dry it up.